src/ci/docker/mingw-check/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN sh /scripts/sccache.sh
 COPY mingw-check/validate-toolstate.sh /scripts/
 
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
-ENV SCRIPT python2.7 ../x.py check --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu && \
+ENV SCRIPT python2.7 ../x.py test src/tools/expand-yaml-anchors && \
```

Comment:
Maybe this should happen.. last? We don't want to waste time up front on this builder for things that rarely fail I think.

Reply:
I don't think it matters, building and running `expand-yaml-anchors` takes 10 seconds on CI. I'd also prefer to have "you're running with the wrong CI configuration" show up as soon as possible.